### ISSUE TITLE
Rework Choosing a Decision Review Option page to take content out of table

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#40464]
## Issue Description
To make sure VA.gov is easily read by screen readers and works well on mobile, we need to use tables only when the information can't be displayed in any other way. Oftentimes, information in a table can be put into a list form instead. 

Currently, we have a table on our Resources and Support page [Choosing a decision review option](https://www.va.gov/resources/choosing-a-decision-review-option/) that we want to change to a different format. 

The content team needs to assess the best way to present this information outside of a table while retaining all the information.    

## Tasks
- [x] Assess the right format for the content (Aliyah)
- [x] Create word doc with new content (Aliyah)
- [x] High-level edit (Laura)
- [x] Copy edit (Megan)

## Acceptance Criteria
- [x] Updated page is ready for OCTO-DE review

**Comment by laurawillwerth** [comment]
Hi @aliyahblackmore , thanks for working through these updates with me! Can you take a look at the latest version in Drupal and see if you have any questions? Then you can pass to Megan. (I'm also updating your Google doc with some notes for when we pass to Beth.)

**Comment by aliyahblackmore** [comment]
Hi @megzehn - this page is ready for you to review! Our google doc is [here](https://docs.google.com/document/d/1idv_d9nmAy8bqVZ1gkx8bUnJxWMRahey3XHLBMJ23nU/edit)

- [ ] [Choosing a decision review option](http://preview-prod.vfs.va.gov/preview?nodeId=23290)

**Comment by megzehn** [comment]
Thanks! I made some minor tweaks, and I will pass it on to Beth for a final review.

**Comment by RLHecht** [comment]
I reworked the tasks and acceptance criteria for this ticket so it's complete when our team is ready to hand this to Beth for review. Closing this and creating a new ticket for Beth's review and sending to stakeholders.

**Comment by bethpottsVADEPO** [comment]
@aliyahblackmore @laurawillwerth @megzehn Thanks for your work on this! Based on comments from the stakeholders, one problem they were having that the original chart solved was people thinking there's a progression of some sort (like start by filing a supplemental claim, then if that's not successful, file Higher-Level Review, then a Board Appeal). So what you have now wouldn't address that issue. Could you try an approach more like this page: http://preview-prod.vfs.va.gov/preview?nodeId=44718

In other words, you'd have H2s that would be something like (please improve if you have better phrasing!):
Reason for your decision review
How long each type of decision review takes
etc.

And then 3 H3s within each H2:
Supplemental Claim
Higher-Level-Review
Board Appeal

Thanks for reworking, and let me know if you want to chat about it while you're in progress!

**Comment by bethpottsVADEPO** [comment]
@RLHecht Thanks for your patience as we figure out where I put feedback to the team if there's a need to rework. I'm fine with doing that either in the content team ticket or in my OCTO review ticket. But it feels like we could use this original ticket for refining/feedback on the work since I'm trying to do my pass within the same sprint. Then if we get close to the end of the sprint, we decide if the original ticket can be closed. Can we wrap up this work tomorrow and still consider it work done (for the most part) in the sprint that's about to end?

**Comment by RLHecht** [comment]
That makes sense @bethpottsVADEPO We still have until next Tuesday for the end of this sprint so there is still plenty of time. For this one when we're ready to send to stakeholders I think we can still use the ticket I set up for that, which is #41159. Since it's possible they won't have feedback for us until after the sprint is closed, we'll be able to close out this ticket and continue the work in the next one.

**Comment by aliyahblackmore** [comment]
Just noting here: after our 5/11 stand up I was in touch with @laurawillwerth to ask if she would like me to review any of the notes she has taken following Beth's feedback.  More soon!

**Comment by bethpottsVADEPO** [comment]
@RLHecht @laurawillwerth @aliyahblackmore @megzehn In case it's helpful as you rework this page, here's the original comparison doc I mentioned on our call (this is what we created the current chart from):
https://www.vba.va.gov/pubs/forms/VBA-20-0998-ARE.pdf

**Comment by aliyahblackmore** [comment]
Adding a status update here -> I am going to spend the next minutes working on one option for the rework of this page based on Beth's above feedback and suggestion.

**Comment by aliyahblackmore** [comment]
Hi @laurawillwerth - I finished up a rework of our page, based on Beth's suggestion and feedback from above. Google doc is [here](https://docs.google.com/document/d/1pmbokwUQyOEalOgJ8c36wVEtPpr2ECIW/edit) (with notes on some small placement/text changes).  Passing this to you, Laura, to review. Feel free to add any additional options (that you may have been thinking of) to this Google Doc!

**Comment by laurwill** [comment]
Thanks Aliyah and Katherine for thinking through the header options here! And thanks @bethpottsVADEPO for your feedback — I've attached a Word doc here with 2 updated options for the H2s. Let us know what you think!

Also noting an update from stakeholder re: offering more guidance as to the use case for each review type: "All three options need to be offered and presented as equally viable, without any language that would bias Veterans to select one over another. One of the features of the law (AMA) was to offer Veterans a choice on how they want their claim to proceed if they disagree with their decision. VA systems and communications shouldn’t influence or incentivize Veteran behavior."

[40464 - Choosing a decision review.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/8703583/40464.-.Choosing.a.decision.review.docx)
